documentation/4.0/content/managing-domains/model-in-image/usage.md

```diff
@@ -167,6 +175,12 @@ The following Domain fields are specific to Model in Image domains.
 | `configuration.model.modelHome`              | Optional. Location of the WDT model home, which can include model YAML files, `.properties` files, and application `.zip` archives. Defaults to `/u01/wdt/models` if no [Auxiliary Images]({{<relref "/managing-domains/model-in-image/auxiliary-images" >}}) are configured, and to `/aux/models` otherwise.|
 | `configuration.model.wdtInstallHome`         | Optional. Location of the WDT installation. Defaults to `/u01/wdt/weblogic-deploy` if no [Auxiliary Images]({{<relref "/managing-domains/model-in-image/auxiliary-images" >}}) are configured, and to `/aux/weblogic-deploy` otherwise.|
 
+{{% notice note %}}
+If you set `modelHome` and `wdtInstallHome` to a non-default value,
+then the operator will ignore WDT model and installation files
```

Comment:
Should this say "domain" instead of the operator similar to line 116 in `documentation/4.0/content/managing-domains/model-in-image/auxiliary-images.md`?

Reply:
Probably. That's the original wording copied from before these changes. I changed it to domain in other locations IIRC.

Since the pull is already merged, I will create a new pull.